src/apps/loyalty/analytics.ts

```diff
@@ -0,0 +1,79 @@
+import { extend, pick } from "lodash"
```

Comment:
The idea is that this is a 'base' file which client-side can be included, with the `initAnalytics()` method setting up the standard boilerplate.